src/ocaml-output/Makefile

```diff
@@ -90,18 +88,19 @@ endif
 	              | cat -s >> $@
 
 # https://stackoverflow.com/questions/38294095/ocaml-how-to-solve-findlib-warnings-of-multiple-cmis
-fstar.exe: export OCAMLFIND_IGNORE_DUPS_IN = $(shell ocamlfind query compiler-libs)
-fstar.exe: $(GENERATED_FILES)
+../../bin/fstar.exe: export OCAMLFIND_IGNORE_DUPS_IN = $(shell ocamlfind query compiler-libs)
+../../bin/fstar.exe: $(GENERATED_FILES)
 	$(OCAMLBUILD) main.native src/ocaml-output/FStar_Syntax_Syntax.inferred.mli
-	cp -f _build/src/fstar/ml/main.native ../../bin/$@
-	cp -f _build/src/fstar/ml/main.native ../../bin/fstar.ocaml
+	./copy_if_diff.sh _build/src/fstar/ml/main.native $@
+	./copy_if_diff.sh _build/src/fstar/ml/main.native ../../bin/fstar.ocaml
```

Comment:
why do you need copy_if_diff.sh? ocamlbuild, insofar as I understand it, should, now that the hash of FStar_Version.ml hasn't changed, say "not rebuilding fstar.exe"? and then you could just make this a copy, regardless?

Reply:
that would still change the mtime of the destination file

Reply:
Ok, I see what you mean ... I think. Pushed an update